core/src/main/java/org/apache/accumulo/core/util/format/Formatter.java

```diff
@@ -22,6 +22,10 @@
 import org.apache.accumulo.core.data.Key;
 import org.apache.accumulo.core.data.Value;
 
+/**
+ * @deprecated since 2.0.0. Internal use only. Use TODO: provide SPI option?
```

Comment:
This is already is in a non-public API package. No need to deprecate. It can just be removed.

Reply:
This class only existed to support a shell option, so even though not public API it was advertised to users as something they should use.

Reply:
Ah OK.  I will keep it deprecated then.